.golangci.yml

```diff
@@ -16,7 +16,6 @@ issues:
 linters:
   disable-all: true
   enable:
-    - goconst
```

Comment:
what breaks if we don't remove this? Should we have an issue to put it back in a separate PR? I see that we have it in icm-contracts specific `.golangci.yml`?